v0/destinations/active_campaign/data/ACIdentify.json

```diff
@@ -1,21 +1,26 @@
 [
   {
     "destKey": "email",
-    "sourceKeys": ["traits.email", "context.traits.email", "properties.email"],
+    "sourceKeys": "email",
+    "sourceFromGenericMap": true,
     "required": true
   },
   {
     "destKey": "phone",
-    "sourceKeys": ["traits.phone", "context.traits.phone", "properties.phone"],
+    "sourceKeys": "phone",
+    "sourceFromGenericMap": true,
     "required": false
   },
   {
-    "destKey": "fieldValues",
-    "sourceKeys": [
-      "traits.fieldValues",
-      "context.traits.fieldValues",
-      "properties.fieldValues"
-    ],
+    "destKey": "firstName",
+    "sourceKeys": "firstName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "lastName",
+    "sourceKeys": "lastName",
+    "sourceFromGenericMap": true,
     "required": false
```

Comment:
We shouldn't remove this as during the syncContact function call fieldvalues will not be there.

Reply:
We are mapping it separately in transform.js L407

Reply:
Yeah, that is there, but before we were sending the fieldValues during the first call but now during the syncContact call fieldValues will not be there.

Reply:
Earlier it was wrongly mapped we were not sending actually

Reply:
Okay then.